packages/client/e2e/credit.spec.ts

```diff
@@ -172,7 +172,7 @@ test.describe("Credit", () => {
 
       await expect(rfqsPage.getByTestId("quotes").first()).toContainText(
         "$100",
-        { timeout: ElementTimeout.NORMAL },
+        { timeout: ElementTimeout.LONG },
```

Comment:
30s seems like a heck of a long time to wait for the quote ..

Reply:
especially as we are providing it above .. so within 5 seconds at the outside we should be done (really)

Reply:
agree, will update to 5s `Element.Timeout.AGGRESSIVE`